dotnet/src/SemanticKernel.Abstractions/AI/FunctionChoiceBehaviors/NoneFunctionChoiceBehavior.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System.Collections.Generic;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Represents <see cref="FunctionChoiceBehavior"/> that provides either all of the <see cref="Kernel"/>'s plugins' functions to the AI model to call or specific ones.
+/// This behavior instructs the model not to call any functions and only to generate a user-facing message.
+/// </summary>
+/// <remarks>
+/// Although this behavior prevents the model from calling any functions, the model can use the provided function information
+/// to describe how it would complete the prompt if it had the ability to call the functions.
+/// </remarks>
+internal sealed class NoneFunctionChoiceBehavior : FunctionChoiceBehavior
+{
+    /// <summary>
+    /// List of the functions to provide to AI model.
+    /// </summary>
+    private readonly IEnumerable<KernelFunction>? _functions;
+
+    /// <summary>
+    /// Initializes a new instance of the <see cref="NoneFunctionChoiceBehavior"/> class.
+    /// </summary>
+    /// <param name="functions">
+    /// Functions to provide to AI model. If null, all <see cref="Kernel"/>'s plugins' functions are provided to the model.
+    /// If empty, no functions are provided to the model, which is equivalent to disabling function calling.
```

Comment:
None already disabled function call, suggest updating to:

```suggestion
    /// If empty, no function information will be provided to the model.
```

Reply:
Done.